packages/Utils/src/DTK_Core.hpp

```diff
@@ -0,0 +1,28 @@
+/****************************************************************************
+ * Copyright (c) 2012-2017 by the DataTransferKit authors                   *
```

Comment:
I think we will want to have one main header eventually for C where everything is included for the user